content/applications/inventory_and_mrp/maintenance/maintenance_reporting.rst

```diff
@@ -0,0 +1,295 @@
+:guilabel:`Measures` and :guilabel:`Insert in Spreadsheet` buttons.
+
+.. image:: maintenance_reporting/maintenance-reporting-graph-view-types.png
+   :align: center
+   :alt: Graph type icons on Graph view page.
+
+There are three different types of graphs available to users to view the data:
+
+- :guilabel:`Bar Chart`: the data is displayed in a bar chart.
+- :guilabel:`Line Chart`: the data is displayed in a line chart.
+- :guilabel:`Pie Chart`: the data is displayed in a pie chart.
+
+Click the :icon:`fa-bar-chart` :guilabel:`(bar chart)` icon to switch to a bar chart. When viewing
+the data as a :guilabel:`Bar Chart` graph, the data can be formatted in the following ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+Click the :icon:`fa-line-chart` :guilabel:`(line chart)` icon to switch to a line chart. When
+viewing the data as a :guilabel:`Line Chart` graph, the data can be formatted in the following ways:
+
+- :guilabel:`Stacked`: the data is stacked on the graph.
+- :guilabel:`Cumulative`: the data is increasingly accumulated.
+- :guilabel:`Descending`: the data is displayed in descending order.
+- :guilabel:`Ascending`: the data is displayed in ascending order.
+
+Click the :icon:`fa-pie-chart` :guilabel:`(pie chart)` icon to switch to a pie chart. When viewing
+the data as a :guilabel:`Pie Chart` graph, all relevant data is displayed by default, and no
+additional formatting options are available.
+
+Pivot view
+----------
+
+Click the :icon:`oi-view-pivot` :guilabel:`(pivot view)` icon to change to pivot view.
+
+With the :guilabel:`Pivot` view selected, maintenance requests are displayed in a pivot table, and
+can be customized to show different data metrics.
+
+To add more data to the pivot table, click the :guilabel:`Measures` button to reveal a drop-down
+menu. By default, :guilabel:`Count` is selected. Additional options to add to the table are
+:guilabel:`Additional Leaves to Plan Ahead`, :guilabel:`Duration`, and :guilabel:`Repeat Every`.
+
+.. image:: maintenance_reporting/maintenance-reporting-pivot-measures.png
+   :align: center
+   :alt: Measures options on Pivot view page.
+
+To the right of the :guilabel:`Insert in Spreadsheet` button are three buttons:
+
+- :guilabel:`Flip axis`: the x and y axis of the pivot data table flip.
+- :guilabel:`Expand all`: all the available rows and columns of the pivot data table expand fully.
+- :guilabel:`Download xlsx`: the pivot data table is downloaded as an .xlsx file.
+
+Kanban view
+-----------
+
+Click the :icon:`oi-view-kanban` :guilabel:`(Kanban view)` icon to change to Kanban view.
+
+With the :guilabel:`Kanban` view, all open maintenance requests are displayed in Kanban-style
+columns, in their respective stages of the maintenance process.
```

Comment:
By default (in my runbot) the Repaired and Scrap are folded- might want to add that, if that is the default?

Reply:
Added that in! Suggestion implemented ✅